.github/workflows/e2e-reusable.yml

```diff
@@ -0,0 +1,135 @@
+name: Reusable e2e workflow
+
+on:
+  workflow_call:
+    inputs:
+      branch:
+        description: 'GitHub branch to test.'
+        required: false
+        type: string
+        default: 'parallel_e2e'
+      user:
+        description: 'User who kicked this off.'
+        required: false
+        type: string
+        default: 'schedule'
+      spec:
+        description: 'Specify specs.'
+        required: false
+        default: 'e2e/*'
+        type: string
+      run-env:
+        description: 'Node env version to run tests with.'
+        required: false
+        default: 'browsers:node16.18.0-chrome90-ff88'
+        type: string
+      record:
+        description: 'Record test run.'
+        required: false
+        default: true
+        type: boolean
+      parallel:
+        description: 'Run tests in parallel.'
+        required: false
+        default: true
+        type: boolean
+      containers:
+        description: 'Number of containers to run tests in.'
+        required: false
+        default: '[1, 2, 3, 4, 5, 6]'
+        type: string
+    secrets:
+      CYPRESS_RECORD_KEY:
+        description: 'Cypress record key.'
+        required: true
+
+jobs:
+  install:
+    runs-on: ubuntu-latest
+    container:
+      image: cypress/${{ inputs.run-env }}
+      options: --user 1001
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: n8n-io/n8n
+          ref: ${{ inputs.branch }}
+
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2.2.4
+        with:
+          run_install: true
+
+      - name: Echo spec input
+        run: echo ${{ inputs.spec }}
+
+      - name: Cache pnpm modules
+        uses: actions/cache@v3
+        with:
+          path: |
+            /github/home/.cache
+            /github/home/.pnpm-store
+            ./node_modules
+            ./packages
+          key: ${{ github.sha	}}-${{ inputs.run-env }}-e2e-modules
+
+      - name: Cypress build
+        uses: cypress-io/github-action@v5
+        with:
+          # Disable running of tests within install job
+          runTests: false
+          install: false
+          build: pnpm build
+
+      - name: Cypress install
+        run: pnpm cypress:install
+
+  testing:
```

Comment:
Can you set a name here to display the job name as `Spec: "${{ inputs.spec }}"` instead of `testing (x)`?

Reply:
not here no.. though there's a github status thing that would show the specs